### PR TITLE
Add neo-scan-sync to CLI

### PR DESCRIPTION
**Pull request by nunojusto**
## Problem

Add neo-scan-sync to CLI
Solving issue https://github.com/CityOfZion/neo-local/issues/72

## Solution

Adding neo-scan-sync

## Checklist

- [x] I have ran the tests locally.
- [x] I branched off of the `develop` branch and not `master`.
- [x] I did not change the `VERSION` file.

### CODE REVIEW

File: cli/stack/neo-scan-sync.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+				"DB_DATABASE=neoscan_prodv",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4002/tcp": {},
+			},
+		},
+		DependsOn: []string{"postgres", "neo-privatenet", "notifications-server"},
+		HostConfig: &container.HostConfig{
+			Links:      []string{"neo-privatenet:30333", "neo-privatenet:30334", "neo-privatenet:30335", "neo-privatenet:30336"},
```
[the remark this answers is not in the document]
Comment (nunojusto) [reply cli/stack/neo-scan-sync.go]: done

File: cli/stack/neo-scan-sync.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+				"DB_USERNAME=postgres",
+				"DB_PASSWORD=postgres",
+				"DB_DATABASE=neoscan_prodv",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4002/tcp": {},
+			},
+		},
+		DependsOn: []string{"postgres", "neo-privatenet", "notifications-server"},
```
[the remark this answers is not in the document]
Comment (nunojusto) [reply cli/stack/neo-scan-sync.go]: done

File: cli/stack/neo-scan-sync.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+func NewNeoScanSync() Service {
+	return Service{
+		Author: "cityofzion",
+		ContainerConfig: &container.Config{
+			Env: []string{
+				"NEOSCAN=neo-scan-api:4000",
+				"NEO_NOTIFICATIONS_SERVER='http://notifications-server:8080/v1'",
+				"NEO_SEEDS='http://neo-privatenet:30333;http://neo-privatenet:30334;http://neo-privatenet:30335;http://neo-privatenet:30336'",
+				"DB_HOSTNAME=postgres",
+				"DB_USERNAME=postgres",
```
Comment (revett) [inline cli/stack/neo-scan-sync.go]: These need to be A-Z

File: cli/stack/neo-scan-sync.go [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+				"DB_DATABASE=neoscan_prodv",
+				"REPLACE_OS_VARS=true",
+			},
+			ExposedPorts: map[nat.Port]struct{}{
+				"4002/tcp": {},
+			},
+		},
+		DependsOn: []string{
+			"postgres",
+			"neo-privatenet",
```
Comment (revett) [inline cli/stack/neo-scan-sync.go]: Also A-Z

Comment (nunojusto) [reply cli/stack/neo-scan-sync.go]: done

File: cli/stack/neo-scan-sync.go (hunk 3 above)
Comment (nunojusto) [reply cli/stack/neo-scan-sync.go]: done

File: cli/stack/neo-scan-sync.go [hunk 5]
```diff
@@ -0,0 +1,45 @@
+package stack
+
+import (
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/go-connections/nat"
+)
+
+// NewNeoScanSync creates a new service for the egistry.gitlab.com/cityofzion/neo-scan/sync:latest image.
```
Comment (revett) [inline cli/stack/neo-scan-sync.go]:
```suggestion
// NewNeoScanSync creates a new service for the
// registry.gitlab.com/cityofzion/neo-scan/sync:latest image.
```

**Comment by revett** [comment]
@nunojusto Can you resolve conflicts please?

**Pull request merged**